index.js

```diff
@@ -91,6 +92,21 @@ module.exports = function(input, inputMap) {
 			});
 			return;
 		}
+
+		if (params.includeModulePaths) {
+			// Ensure module paths include full path from current working directory
+			var pwd = process.cwd();
+			if (context.substring(0, pwd.length) === pwd) {
```

Comment:
I just looked again. Extracting it to line 20 isn't possible. It's using the `map` and `context` arguments from `processMap()`. It's similar to the callbacks passed to `async.map()` at line 63.